processing/src/main/java/io/druid/segment/column/BitmapIndexSeeker.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Druid - a distributed column store.
+ * Copyright 2012 - 2015 Metamarkets Group Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.druid.segment.column;
+
+import io.druid.segment.data.IndexedInts;
+
+public interface BitmapIndexSeeker
```

Comment:
Can we clarify what the assumptions are in terms of access order, and what the contract for seek is?